frontend/src/pages/BankTransactions.jsx

```diff
@@ -20,8 +26,39 @@ const BankTransactions = () => {
         perPage,
     });
 
-    const handleFileUpload = (file) => {
-        console.log("File uploaded:", file);
+    const queryClient = useQueryClient();
```

Comment:
Issue: Mutation logic was duplicated in the component instead of using the existing useBucketFileUpload hook.
Fix: Use useBucketFileUpload from queries/uploadedFiles.js. Keep the mutation logic in the hook; in the component, just call the hook and use it. This removes duplication and matches the pattern used elsewhere (like useCreateUserStatus).
Changes:
Replaced inline useMutation with useBucketFileUpload
Removed unnecessary imports
Component now only handles UI; mutation logic stays in the hook